README.md

```diff
@@ -48,3 +48,4 @@ Note that:
 * Pokémon 20 Questions ([source](https://www.github.com/ashbaldry/Pokemon_20Q), [app](https://ashbaldry.shinyapps.io/Pokemon_20Q/), [blog](https://community.rstudio.com/t/shiny-contest-submission-pokemon-20-question-game/25726)) by [Ashley Baldry](https://ashbaldry.github.io/)
 * Shiny Decisions ([source](https://github.com/pedrocoutinhosilva/shiny.decisions), [app](https://sparktuga.shinyapps.io/ShinyDecisions/), [blog](https://appsilon.com/is-it-possible-to-build-a-video-game-in-r-shiny/)) by [Pedro Coutinho Silva](https://www.pedrocsilva.com/)
 * Wordle ([source](https://github.com/wch/shiny-wordle), [app](https://winston.shinyapps.io/wordle/), [video playlist](https://www.youtube.com/playlist?list=PL9HYL-VRX0oQnWIeY_ydYBdU76iQ-tchU)) by [Winston Chang](https://www.rstudio.com/authors/winston-chang/)
+* Whack-A-Mole ([source](https://github.com/tomicapretto/whackamole))
```

Comment:
Thank you Tomás! Could you also add to the end of this line your name and a link to your website, Twitter, or similar? Like: 'by [<your name>](<link to website>)'.